week_1/README.md

```diff
@@ -0,0 +1,33 @@
+# Assignments - Week 1
+
```

Comment:
Also add lines that ask them to try these things hands on. Provide links to an online editor as well.